src/ilcdlib/mapping/impacts.py

```diff
@@ -48,7 +48,7 @@ class ImpactsKeywordToOpenIdMapper(KeyValueMapper[str]):
     KV = {
         "gwp-biogenic": ["biogenic"],
         "gwp-luluc": ["luluc"],
-        "gwp-nonCO2": ["CO2"],
+        "gwp-nonCO2": ["CO2", "non fossil"],
```

Comment:
What is an example of the object we want to map with `"non fossil"` keyword?

Reply:
It's hard to find now, but it was once listed as 'non-CO2 GWP' using the term 'non-fossil' instead of CO2